daeun/lv1/2016년.js

```diff
@@ -0,0 +1,4 @@
+function solution(a, b) {
+    const week = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']
+    return week[new Date(`2016-${a}-${b}`).getDay()];
+}
```

Comment:
```suggestion
function solution(a, b) {
    const week = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']
    return week[new Date(`2016-${a}-${b}`).getDay()];
}
```
템플릿 리터럴을 활용해서 코드를 더 간단하게 만들 수 있는 풀이법 잘 보고 배워갑니닷!

Reply:
코멘트 감사합니다! 👍